src/app/(pages)/exam.tsx

```diff
@@ -26,23 +30,20 @@ export default function ExamDetail(): JSX.Element {
                     value: formatFriendlyDateTime(
                         exam?.date as unknown as string
                     ),
-                    disabled: true,
                 },
 
                 {
                     title: t('pages.exam.details.room'),
                     value: exam?.rooms,
-                    disabled: true,
                 },
                 {
                     title: t('pages.exam.details.seat'),
                     value: exam?.seat,
-                    disabled: true,
                 },
                 {
                     title: t('pages.exam.details.tools'),
                     value: exam?.aids?.join(', '),
-                    disabled: true,
+                    layout: 'column',
```

Comment:
cool new feature, but it looks wrong with short values like 'Taschenrechner'. You could try to set the type conditional